packages/components/Atoms/Global/utils/helpers.js

```diff
@@ -81,4 +81,14 @@ function _isTelOrEmailUrl (url) {
   return false
 }
 
+export const truncateText = (text, stop = 150, clamp) => {
```

Comment:
As this is now a duplicate of the function in packages/ripple-nuxt-tide/lib/core/tide-helper.js, is there any benefit in reusing the same code? E.g. updating nuxt-tide to use rpl-global?

Reply:
Hi @alan-cole ,
it is updated to new path now.